### DISCUSSION TITLE
No option to cancel "import" command.

### DISCUSSION THREAD

**Discussion by Riyyi** [ImageMagick/ImageMagick#3717] [Help]
I tried searching through the documentation, the project issues and the discussions but couldn't find anything.

How do you cancel the mouse selection after starting it if the ```import``` command is run from a script (so you can't Ctrl-C)?
```import.c``` only mentions this:
```c++
%  The target window can be specified by id, name, or may be selected by
%  clicking the mouse in the desired window. If you press a button and then
%  drag, a rectangle will form which expands and contracts as the mouse moves.
%  To save the portion of the screen defined by the rectangle, just release
%  the button.
```

**Comment by urban-warrior** [comment]
Your only choice is to click on something or click-hold-drag.

**Reply by Riyyi** [reply]
That's unfortunate, guess I'll install maim then.

**Reply by urban-warrior** [reply]
If you simply want a screenshot without user intervention:
```
import -window root screenshot.png
```

**Reply by unrealapex** [reply]
I think what OP wants is a key that could be used to abort the screenshot. Maim uses <kbd>Escape</kbd>.
